File: 1920. Build Array from Permutation/solution.py
class Solution(object):
  def buildArray(self, nums):
    """
    :type nums: List[int]
    :rtype: List[int]
    """
    result = [0] * len(nums)
    for index, num in enumerate(nums):
      result[index] = nums[num]
      
    return result

File: 1920. Build Array from Permutation/solution.cpp
class Solution {
public:
  vector<int> buildArray(vector<int>& nums) {
    vector<int> result(nums.size(), 0);
    for (int index = 0; index < nums.size(); index++){
      int num = nums[index];
      result[index] = nums[num];
    }

    return result;
  }
};